src/Dashboard/Grid.php

```diff
@@ -1246,7 +1263,7 @@ public function getAllDasboardCards($force = false): array
                     'State',
                     'Entity',
                     'Manufacturer',
-                    'Location',
+                    LocationFilter::getId(),
```

Comment:
```suggestion
                    'Location',
```
I think this one should be reverted, does not seem related to filters.